tornado/platform/common.py

```diff
@@ -84,8 +84,10 @@ def consume(self):
                 result = self.reader.recv(1024)
                 if not result:
                     break
-        except (IOError, socket.error):
-            pass
+        except (IOError, socket.error) as e:
+            if e.errno != errno.EWOULDBLOCK:
```

Comment:
Are EWOULDBLOCK and EAGAIN equivalent on windows? This should probably use the _ERRNO_WOULDBLOCK constant from iostream.py.